balancer-js/examples/exactTokensInJoinPool.ts

```diff
@@ -0,0 +1,54 @@
+import dotenv from 'dotenv';
```

Comment:
Should we rename example to just joinPool? Thinking we want to keep it simple and don’t want to make it seem like there’s more options to consider unless they are implemented?